packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -132,6 +133,10 @@ contract VotingV2 is
     // Max length in bytes of ancillary data that can be appended to a price request.
     uint256 public constant ANCILLARY_BYTES_LIMIT = 8192;
 
+    // Hard coded address of previous voting contract. Used to claim rewards from previous contract, if unclaimed.
+    VotingAncillaryInterface public constant v1Voting =
+        VotingAncillaryInterface(0x8B1631ab830d11531aE83725fDa4D86012eCCd77);
```

Comment:
Why not make this an immutable variable that's set in the constructor? That would allow us to unit test this functionality.